<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>debugForm</class>
 <widget class="QWidget" name="debugForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTextBrowser" name="debug"/>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="showFatal">
       <property name="text">
        <string>Show Fatal Messages</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="showCritical">
       <property name="text">
        <string>Show Critical Messages</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="showWarning">
       <property name="text">
        <string>Show Warning Messages</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="showInfo">
       <property name="text">
        <string>Show Info Messages</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="showDebug">
       <property name="text">
        <string>Show Debug Messages</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="clearAll">
       <property name="text">
        <string>Clear All</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
